doc/man3/OSSL_PARAM.pod

```diff
@@ -78,6 +88,11 @@ setting parameters) or shall (when requesting parameters) be stored,
 and C<data_size> is its size in bytes.
 The organization of the data depends on the parameter type and flag.
 
+When the C<OSSL_PARAM> is used as a parameter descriptor, C<data>
+should be ignored.
+If C<data_size> may be zero, it means that an arbitrary data size is
+accepted, otherwise it specifies the maximum size allowed.
```

Comment:
Hmmm, is a minimum size desirable as well?

Thinking a fixed sized key, or a version string that needs to be a least _n_ characters long.

Reply:
I probably should reword this.  This is rather an indication of *expected* size, i.e. how large the data space should be (in a `_set_params` scenario, it's what the setter expects to get, in a `_get_params` scenario, it's how large the caller's buffer should be).  This is a purely technical aspect.

Minimum size, on the other hand, is more of a rule to be or not to be applied.  It doesn't really have a *technical* impact, and I'd rather have rules governed by code and rule breakage signalled back with an error.